### ISSUE TITLE
`no-bare-strings` false-negative

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/ember-template-lint#2305]
```hbs
<div>
  {{if true "Yes" "No"}}
</div>
```

This should lead to a `no-bare-strings` warning but it doesn't.
